src/jit/codegenxarch.cpp

```diff
@@ -2631,159 +2605,63 @@ CodeGen::genCodeForTreeNode(GenTreePtr treeNode)
     }
 }
 
-//------------------------------------------------------------------------
-// genStoreRegisterReturnInLclVar: This method handles storing double register return struct value to a 
-// local homing stack location.
+//----------------------------------------------------------------------------------
+// genMultiRegStoreToLocal: store multi-reg return value of a call node to a local
 //
 // Arguments:
-//    treeNode  - the tree which should be homed in local frame stack location.
+//    treeNode  -  Gentree of GT_STORE_LCL_VAR
 //
 // Return Value:
-//    For System V AMD64 sistems it returns true if this is a struct and storing of the returned
-//    register value is handled. It returns false otherwise.
-//    For all other targets returns false.
-
-bool
-CodeGen::genStoreRegisterReturnInLclVar(GenTreePtr treeNode)
+//    None
+//
+// Note
+//    Right now this is implemented only for x64 Unix.
```

Comment:
Although it is stated in the description, you should explicitly state the assumption/pre-condition that the child of the store is a call node.